src/app/pages/questions/components/finish/finish.component.ts

```diff
@@ -68,4 +71,23 @@ export class FinishComponent implements OnChanges {
   toggleChanged(event) {
     this.completedInClinic = event
   }
+
+  getEtaText(progress) {
+    const duration = getMinutes({ milliseconds: Date.now() - this.startTime })
+    return (
+      'About ' + (duration * (100 - progress)) / progress + ' minutes remaining'
+    )
+  }
+
+  getProgressBarStyle(progress) {
+    return progress >= 1
+      ? `inset 500px 0 0 -50px var(--cl-primary-60)`
+      : `inset ${progress * 400}px 0 0 -50px var(--cl-primary-60)`
+  }
+
+  getFinishButtonText(progress) {
+    return progress >= 1
+     ? this.localization.translateKey(LocKeys.SETTINGS_WAIT_ALERT) + '...'
+     : this.localization.translateKey(LocKeys.BTN_DONE)
```

Comment:
should this be the other way around, i.e when progress is over 1 (or complete) then show the Done text?

Reply:
Ah yes made the changes.